### DISCUSSION TITLE
[Feature Request] Allow README-style markdown homepage for Organisation

### DISCUSSION THREAD

**Discussion by benhutchison** [community/community#7443] [Other Feature Feedback, Questions, & Ideas] [answered]
Organisation landing pages currently showcase Pinned Repositories. 

This request is for ability to add a page of user-customised markdown to the landing page rather than, or above, repo links. This could be useful to:
- Describe the overall organisation
- Quick links to docs, projects, issues
- Identify people and roles
- Describe processes for people working in the org

Increasingly github is about so much more than just repositories, and markdown would offer more flexibility in how Orgs could present themselves to users.

**Comment by TheSleepyAwoo** [comment, marked as answer]
You can read about how to add an organisation readme here: [https://docs.github.com/en/organizations/collaborating-with-groups-in-organizations/customizing-your-organizations-profile](https://docs.github.com/en/organizations/collaborating-with-groups-in-organizations/customizing-your-organizations-profile). 

You can see an example of an organisation readme at GitHub's organisation: [https://github.com/github](https://github.com/github).
